test/src/main/java/org/apache/accumulo/test/ScanServerMultipleScansIT.java

```diff
@@ -190,7 +191,8 @@ public void testMultipleScansDifferentTablets() throws Exception {
         final int threadNum = i;
         var future = executor.submit(() -> {
           try {
-            latch.await();
+            startLatch.countDown();
+            startLatch.await();
           } catch (InterruptedException e1) {
             fail("InterruptedException waiting for latch");
```

Comment:
```suggestion
            fail("InterruptedException waiting for startLatch");
```
for consistency with other test